src/components/Claims/Claims.tsx

```diff
@@ -0,0 +1,287 @@
+import { useChannel } from '@ably-labs/react-hooks'
+import { useEffect, useReducer, useState } from 'react'
+import { useOutletContext } from 'react-router-dom'
+import { Types } from 'ably'
+import type { ProjectInfo } from '../../Layout'
+import {
+  RefreshIcon,
+  ShieldCheckIcon,
+  UserCircleIcon,
+} from '@heroicons/react/solid'
+import randomWords from 'random-words'
+import JWTUtil from './JWTUtil'
+
+type Message = {
+  author: string
+  content: string
+  timestamp: Date
+  id: string
+  deleted?: boolean
+}
+
+type MessageSendEvent = { type: 'send'; message: Message; id: string }
+type MessageClearEvent = { type: 'clear' }
+type MessageDeleteEvent = { type: 'delete'; [key: string]: any }
+
+type MessageDispatch = MessageSendEvent | MessageClearEvent | MessageDeleteEvent
+
+const Claims = () => {
+  const messageReducer = (
+    state: Message[],
+    action: MessageDispatch
+  ): Message[] => {
+    switch (action.type) {
+      case 'send':
+        action.message.id = action.id
+        return state.concat(action.message)
+      case 'delete':
+        // Delete the message by remapping the message list with the target message deleted
```

Comment:
For consistency across the repo, could we use the comments style from here:
`src/components/AvatarStack/AvatarStack.tsx`

Reply:
Fixed https://github.com/ably-labs/realtime-examples/pull/37/commits/40270d365d6721c75ae07cd6a574e33758a1ed4b